src/app/common/utils/system-notifications/system-notifications.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { IShowNotificationConfig } from './system-notifications.enums';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SystemNotificationsService {
+    public constructor(
+        private readonly snackBar: MatSnackBar,
+      ) { }
+
+      public showNotification({ type, message, dismiss = 'OK' }: IShowNotificationConfig): void {
+        this.snackBar.open(message, dismiss, {
+            panelClass: type,
```

Comment:
From what I can see this is responsible for:
```
Extra CSS classes to be added to the snack bar container.
```
So if we will use `NotificationType.Success` it will add to snackbar css class `success` - but we actually don't do anything about it, we don't change styles of the snackbar anywhere base on this class. So I think we can completely remove it for now.